homeassistant/components/alexa_devices/quality_scale.yaml

```diff
@@ -28,33 +28,31 @@ rules:
   # Silver
   action-exceptions: todo
   config-entry-unloading: done
-  docs-configuration-parameters: todo
-  docs-installation-parameters: todo
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
   entity-unavailable: done
   integration-owner: done
   log-when-unavailable: done
   parallel-updates: done
   reauthentication-flow: todo
-  test-coverage:
-    status: todo
-    comment: all tests missing
+  test-coverage: done
 
   # Gold
   devices: done
-  diagnostics: todo
+  diagnostics: done
   discovery-update-info:
     status: exempt
     comment: Network information not relevant
   discovery:
     status: exempt
     comment: There are a ton of mac address ranges in use, but also by kindles which are not supported by this integration
-  docs-data-update: todo
-  docs-examples: todo
-  docs-known-limitations: todo
-  docs-supported-devices: todo
-  docs-supported-functions: todo
+  docs-data-update: done
+  docs-examples: done
+  docs-known-limitations: done
```

Comment:
known limitations have not been added

Reply:
I don't have further ideas than the warning about MFA.

What do you think we should add ?
For example, so far we have no assurance about non working devices